README.Rmd

```diff
@@ -0,0 +1,90 @@
+---
+output: github_document
+---
+
+```{r, include = FALSE}
+lgr::get_logger("mlr3")$set_threshold("warn")
+lgr::get_logger("bbotk")$set_threshold("warn")
+set.seed(1)
+options(datatable.print.class = FALSE, datatable.print.keys = FALSE)
+```
+
+# bbotk - Black-Box Optimization Toolkit
+
+Package website: [release](https://bbotk.mlr-org.com/) | [dev](https://bbotk.mlr-org.com/dev/)
+
+<!-- badges: start -->
+[![tic](https://github.com/mlr-org/bbotk/workflows/tic/badge.svg?branch=main)](https://github.com/mlr-org/bbotk/actions)
+[![CRAN Status Badge](https://www.r-pkg.org/badges/version-ago/bbotk)](https://cran.r-project.org/package=bbotk)
+<!-- badges: end -->
+
+This package provides a common framework for optimization including
+
+* `Optimizer`: Objects of this class allow you to optimize an object of the class `OptimInstance`.
+* `OptimInstance`: Defines the optimization problem, consisting of an `Objective`, the `search_space` and a `Terminator`.
+   All evaluations on the `OptimInstance` will be automatically stored in its own `Archive`.
+* `Objective`: Objects of this class contain the objective function.
+   The class ensures that the objective function is called in the right way and defines, whether the function should be minimized or maximized.
+* `Terminator`: Objects of this class control the termination of the optimization independent of the optimizer.
+
+Various optimization methods are already implemented e.g. grid search, random search and generalized simulated annealing.
+
+## Installation
+
+CRAN version
+
+```{r, eval = FALSE}
+install.packages("bbotk")
+```
+
+Development version
+
+```{r, eval = FALSE}
+remotes::install_github("mlr-org/bbotk")
+```
+
+## Example
```

Comment:
You could also take the opportunity and present this as  `bbotk in 5 lines of code` while the latter stuff just shows a more detailed version.

Reply:
... and by this I mean the new `bboptimize`.

Reply:
Done